assets/js/src/settings/pages/send_with/other/php_mail_fields.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { Label, Inputs } from 'settings/components';
+import { t, onChange } from 'common/functions';
+import { useSetting } from 'settings/store/hooks';
+import SendingFrequency from './sending_frequency';
+
+export default function PHPMailFields() {
+  const [hostName, setHostName] = useSetting('web_host');
+  const host = hosts[hostName];
+  return (
+    <>
+      <Label title={t('yourHost')} htmlFor="mailpoet_web_host" />
+      <Inputs>
+        <select id="mailpoet_web_host" value={hostName} onChange={onChange(setHostName)}>
+          {Object.values(hosts).map((h) => <option key={h.name} value={h.name}>{h.label}</option>)}
+        </select>
+      </Inputs>
+      <SendingFrequency recommendedEmails={host.emails} recommendedInterval={host.interval} />
+    </>
+  );
+}
+
+const hosts = {
```

Comment:
We have those values already in `lib/Settings/Hosts.php`. Do you think we should re-use them?